### PR TITLE
Allow for specification of organization name when adding term

### PR DESCRIPTION
**Pull request by fgregg**
In Chicago, the City Clerk and the Mayor are separately elected executive officers. They both introduce legislation into City Council. In order to add a post of Mayor or City clerk, it's not sufficient to specify that the primary_org is the 'executive'. This PR lets a scraper specify the name of the organization.

### CODE REVIEW

**Comment by jpmckinney** [comment]
What sort of values would you pass under `primary_org_name`?

**Comment by fgregg** [comment]
The name of the office. So, for Chicago it's "Office of the Mayor" and "Office of the City Clerk" 

For our previous mayor.

```
        daley = Person(name="Daley, Richard M.",
                       primary_org='executive',
                       role='Mayor',
                       primary_org_name='Office of the Mayor')
        daley.add_source('https://chicago.legistar.com/People.aspx')
        yield daley
```

**Comment by jpmckinney** [comment]
Can you add a test in which you use two orgs with the same classification but different name?

**Comment by fgregg** [comment]
Yup.

**Comment by fgregg** [comment]
@jpmckinney this now has the test case you requested.

File: pupa/importers/events.py [hunk 1]
```diff
@@ -55,4 +67,35 @@ def prepare_for_db(self, data):
...
+                    participant['person_id'], 
+                    allow_no_match=True)
+            elif 'organization_id' in participant:
+                participant['organization_id'] = self.org_importer.resolve_json_id(
+                    participant['organization_id'], 
+                    allow_no_match=True)
+
+        for item in data['agenda']:
+            for entity in item['related_entities']:
+                if 'id' in entity :
```
Comment (jpmckinney) [inline pupa/importers/events.py]: Remove space before colon.

Comment (jpmckinney) [inline pupa/importers/events.py]: Actually, you just don't need this if branch at all.

File: pupa/importers/vote_events.py [hunk 2]
```diff
@@ -50,6 +50,10 @@ def get_object(self, vote_event):
 
         return self.model_class.objects.prefetch_related('votes__voter').get(**spec)
 
+    def limit_spec(self, spec):
```
Comment (jpmckinney) [inline pupa/importers/vote_events.py]: Why is this needed? Do events always have a legislative session set?

**Comment by jpmckinney** [comment]
Actually, I'm really confused. Why does this include all of #221?

**Comment by fgregg** [comment]
oops!!! sorry, hold on
